src/config/server.ts

```diff
@@ -70,5 +70,10 @@ export const getServerConfig = () => {
       : 'https://chat-plugins.lobehub.com',
 
     PLUGIN_SETTINGS: process.env.PLUGIN_SETTINGS,
+
+    ENABLE_OAUTH_SSO: process.env.ENABLE_OAUTH_SSO === '1',
```

Comment:
I think there is no need to use `ENABLE_OAUTH_SSO` ,  I perfer to use 
 `ENABLE_OAUTH_SSO: !!process.env.AUTH0_CLIENT_ID`